coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/loader/sql/SqlDatasetDropper.scala

```diff
@@ -9,6 +9,9 @@ import com.rojoma.simplearm.util._
 import java.io.Closeable
 import com.socrata.datacoordinator.truth.DatasetIdInUseByWriterException
 
+//This method was used in a previous implementation of table dropper, where data coordinator put tables first in pending_table_drops
+//Then at a later date, deletes all the datasets in pending_table_drops
+//The To dos refer to that task and are no longer relevant, keeping them around for documentation.
```

Comment:
You can delete all the redundant TODOs. They'll just become a source of confusion at a later date.